website/thaliawebsite/management/commands/migratemedia.py

```diff
@@ -0,0 +1,45 @@
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+import logging
+import os
+import boto3
+from botocore.exceptions import ClientError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
```

Comment:
Maybe at a return here if AWS_BUCKET_NAME == None so the script can't crash if testing locally